### PR TITLE
added logic for scenario to rum multiple times

### PR DESCRIPTION
**Pull request by christina0902**
### **Purpose**
To allow users to decide how many times the heist should be ran.

### **Files Changed**
Added logic to run scenario multiple times based off of user input in `Program.cs`.

### **To Test**
- Confirm program runs correct number of times.
- Confirm each run has a different difficultly level.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -90,19 +95,27 @@
     teamMembers.Add(teamMember);
 }
 
+Console.WriteLine("Please choose number of times to run heist!");
+int trialRunCount = int.Parse(Console.ReadLine());
```
Comment (ztrouy) [inline Program.cs]: We have no error handling, we should probably fix that!

**Review by ztrouy — CHANGES_REQUESTED** [review]
We should have error handling

**Review by ztrouy — APPROVED** [review]
Now works as intended!

**Review by Zavier45 — APPROVED** [review]
good job

**Pull request merged**